service/src/main/resources/static_data/hearingVenues.json

```diff
@@ -1959,6 +1959,16 @@
       "PostCode": "LS1 3BG"
     }
   },
+  {
+    "hearingVenueId": 23,
```

Comment:
this is not enough to have address configured, there is also fpl-ccd-configuration/ccd-definition/FixedLists/CareSupervision/HearingVenue.json which needs to be updated to have court location available in dropdown

Reply:
sorted